docs/_src/api/api/document_classifier.md

```diff
@@ -55,11 +58,29 @@ With this document_classifier, you can directly get predictions via predict()
 |    res["documents"][0].to_dict()["meta"]["classification"]["label"]
  ```
 
+**Usage example at index time:**
+ ```python
+|    ...
+|    converter = TextConverter()
+|    preprocessor = Preprocessor()
+|    document_store = ElasticsearchDocumentStore()
+|    retriever = ElasticsearchRetriever(document_store=document_store)
```

Comment:
I believe we only need the retriever in an indexing pipeline when it is a dense retriever that produces some embeddings. For ElasticsearchRetriever this shouldn't be necessary, but please validate quickly.

Reply:
Yes, removed it.